test-assets/role-manifests/app/tor-invalid-environ-script-reference.yml

```diff
@@ -0,0 +1,42 @@
+# This role manifest should pass validation with no issues
```

Comment:
This comment seems suspicious. (Also the same for the other similar new manifests)

Reply:
Yep